### ISSUE TITLE
Error when creating a wall and other objects that requires me to pick starting point

### ISSUE THREAD

**Issue by lbj** [s-leger/archipack#227]
Archipack version: 2.0.9

Blender version: 2.91

Os version: Ubuntu 20.04 

Description of the issue: When I initiate the wall tool and try to set the starting point of the wall I get the following error code: 

```
Python: Traceback (most recent call last):
  File "/home/lars/.config/blender/2.91/scripts/addons/archipack_20/archipack_snap.py", line 303, in modal
    SnapStore.callback(context, event, 'SUCCESS', self)
  File "/home/lars/.config/blender/2.91/scripts/addons/archipack_20/archipack_wall2.py", line 6511, in sp_init
    res, tM, wall, width, y, z_offset = self.mouse_hover_wall(context, event)
  File "/home/lars/.config/blender/2.91/scripts/addons/archipack_20/archipack_object.py", line 561, in mouse_hover_wall
    res, pt, y, i, o, tM = self.mouse_to_scene_raycast(context, event)
  File "/home/lars/.config/blender/2.91/scripts/addons/archipack_20/archipack_object.py", line 539, in mouse_to_scene_raycast
    vec)
  File "/home/lars/.config/blender/2.91/scripts/addons/archipack_20/archipack_abstraction.py", line 386, in scene_ray_cast
    direction=vec)
TypeError: Scene.ray_cast(): required parameter "depsgraph" not specified

location: <unknown location>:-1
```
Python info: 
PYTHON INTERACTIVE CONSOLE 3.7.7 (default, Jul  9 2020, 12:52:49)  [GCC 9.3.1 20200408 (Red Hat 9.3.1-2)]

**Comment by s-leger** [comment]
Hi Lars,
You should find an e-mail with download link for latest version of archipack (2.3.3).
Download, setup over and restart blender.

**Comment by lbj** [comment]
Hi. 

When I log-in to my gumroad account I see all update notices but the download link (topmost item in screenshot) gives me version 2.0.9. The update logs mention an automatic update feature. Is there a way to trigger an update from within the plugin? 

The request update by e-mail feature gives me the error message "You must enter your gumroad/patreon account e-mail" despite using the correct email. I did however change my email on Gumroad since I purchased the product. I do receive all emails about new updates but no links to the actual update.

![image](https://user-images.githubusercontent.com/1142910/103832868-b6085480-507f-11eb-97dd-65cbef648408.png)
